libs/filabridge/include/filament/MaterialEnums.h

```diff
@@ -27,7 +27,7 @@
 namespace filament {
 
 // update this when a new version of filament wouldn't work with older materials
-static constexpr size_t MATERIAL_VERSION = 12;
+static constexpr size_t MATERIAL_VERSION = 13;
```

Comment:
Do we need to bump the material version? Older materials still work, albeit without lod bias, no?

Reply:
Yes I think it's true that they will work. I added a field to `FrameUniforms` but in this case it shouldn't be a problem.

So... good good point. Maybe we don't need to upgrade the material version.